__tests__/fetcher/fetch-page-data.ts

```diff
@@ -23,3540 +37,491 @@ afterAll(() => {
-              },
-            },
-          },
-        })
-      )
+describe('fetcher: check examples for all supported typenames', () => {
+  test('typename: User', async () => {
+    serverPostUuid({
+      __typename: 'User',
+      id: 1,
     })
-  )
-
-  const response = (await fetchPageData('/de/145590')) as SingleEntityPage
-
-  expect(response.metaData?.title).toEqual('Serlo')
-  expect(response.metaData?.contentType).toEqual('event')
-  expect(response.metaData?.metaDescription).toEqual(
-    '31.08.20 - 25.09.2020, Mo-Fr Digital Learning Academy in MünchenonlineGemeinsame Arbeit an Lerninhalten, Workshops zu verschiedenen Themen …'
-  )
-
-  expect(response.metaData?.metaImage).toEqual(
-    `https://de.${serloDomain}/_assets/img/meta/serlo.jpg`
-  )
-  assertCorrectHorizonDataFormat(response)
-
-  expect(response.cacheKey).toEqual('/de/145590')
-  expect(response.newsletterPopup).toEqual(false)
-  expect(response.kind).toEqual('single-entity')
-  expect(Array.isArray(response.entityData.content)).toEqual(true)
+    const response = (await fetchPageData('/de/1')) as ErrorPage
```

Comment:
```suggestion
    const page = (await fetchPageData('/de/1')) as ErrorPage
```

When I read `response` in a JavaScript project I automatically think of https://developer.mozilla.org/en-US/docs/Web/API/Response so I would use another name -> since the function is named `fetchPageData()` a variable `page` or `pageData` would be better...